package.json

```diff
@@ -106,20 +106,20 @@
     ]
   },
   "devDependencies": {
-    "@types/mocha": "^10.0.7",
-    "@types/node": "^20.14.10",
-    "@types/vscode": "^1.91.0",
+    "@types/mocha": "^10.0.9",
+    "@types/node": "^22.7.5",
+    "@types/vscode": "^1.94.0",
     "@vscode/test-cli": "^0.0.10",
     "@vscode/test-electron": "^2.4.1",
-    "@vscode/vsce": "^2.22.0",
-    "eslint": "^9.6.0",
+    "@vscode/vsce": "^3.1.1",
+    "eslint": "^9.12.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-simple-import-sort": "^12.1.1",
     "eslint-plugin-unused-imports": "^4.0.0",
     "js-yaml": "^4.1.0",
     "prettier": "^3.2.4",
     "ts-loader": "^9.5.1",
-    "typescript": "^5.5.3",
+    "typescript": "~5.5.0",
```

Comment:
note: pinned since eslint does not yet support typescript 5.6.

Ref: https://github.com/typescript-eslint/typescript-eslint/issues/9653.